py/private/py_library.bzl

```diff
@@ -135,8 +135,12 @@ def _make_imports_depset(ctx, imports = [], extra_imports_depsets = []):
     ] + [
         # Add the workspace name in the imports such that repo-relative imports work.
         ctx.workspace_name,
+        ctx.label.workspace_name,
     ]
 
+    # Filter out any empty strings
+    import_paths = [x for x in import_paths if x]
```

Comment:
Is this because `ctx.label.workspace_name` could be an empty string?

Reply:
yes

Reply:
Instead of iterating over the full list in each call to `_make_imports_depset`, could instead this be added conditionally (rather than added then removed)?

Reply:
If that's what you really want, then sure